src/main/72-analytics-database.tf

```diff
@@ -1,3 +1,8 @@
 resource "aws_timestreamwrite_database" "analytics_database" {
   database_name = var.analytics_database_name
+}
+
+resource "aws_timestreamwrite_table" "probing_telemetry" {
```

Comment:
Missing `retention_properties` block. 
The values inside the block should be variables and for now set both arguments to max allowed value.